guides/common/modules/proc_salt-using-content-management.adoc

```diff
@@ -0,0 +1,23 @@
+[id="salt_guide_using_content_management_{context}"]
+= Using {Project}s Content Management
+
+This section provides best practice recommendations on how to provide managed hosts with the required Salt Minion client software using {Project}.
+
+.Procedure
+. Create and synchronize the required products and repositories.
+.. Create a product called `Salt`.
+.. Create a repository within the `Salt` product for each operating system supported by {Project} that you want to install the Salt Minion client software on.
++
+Add the operating system to the name of the repository, for example `Salt for Ubuntu 20.04`.
++
+You can find the _upstream URL_ for the Salt packages on the https://repo.saltproject.io/[official Salt package repository].
+The URL depends on both the Salt version and the operating system, for example `\https://repo.saltproject.io/py3/ubuntu/20.04/amd64/3003/`.
+.. Synchronize the previously created products.
+. Create and publish the required content views.
+.. Create a content view for each repository.
+.. Publish a content view for each of your content views from the previous step.
+. Create a composite content views for each major version of each operating system to make the new content available.
+.. Add each of your operating system specific Salt content views to your main composite content view for that operating system and version.
+.. Publish a new version of the composite content view from the previous step.
+.. Promote the content view from the previous step to your lifecycle environments as appropriate.
+.. Optional: Create activation keys for your composite content view and lifecycle environment combinations.
```

Comment:
I think links would probably be helpful throughout this section. 
```suggestion
.. Optional: Create activation keys for your composite Content View and lifecycle environment combinations.
```

Reply:
Fixed capitalization and added links.